graphql-test-utils/create-graphql-mock-server.js

```diff
@@ -0,0 +1,69 @@
+import { graphql, buildClientSchema } from 'graphql';
+import { addMockFunctionsToSchema } from 'graphql-tools';
+import { formatApolloErrors } from 'apollo-server-errors';
+import loadSchema from './load-schema';
+
+const getRootValue = (operations, operationName, variables) => {
+  if (!operationName || !operations[operationName]) {
+    return {};
+  }
+  const op = operations[operationName];
+  if (typeof op === 'function') {
+    try {
+      return op(variables);
+    } catch (e) {
+      return e; // properly handle dynamic throw new GraphQLError("message")
+    }
+  }
+  return op;
+};
+
+const createGraphqlMockServer = (
+  xhrMock,
+  { mocksByTarget = {}, operationsByTarget = {}, onRequest } = {}
+) => {
+  const processGraphQLRequest = async (targetName, req, res) => {
```

Comment:
Nit: but we always mix casing.

```suggestion
  const processGraphQlRequest = async (targetName, req, res) => {
```